### ISSUE TITLE
OMNA Registry Verification tool

### ISSUE THREAD

**Issue by seanmcilroy29** [OpenMobileAlliance/OMA_LWM2M_Editor#82]
Error code 280 ```Must either be empty if DDF is empty or a valid link to the tool with the URL to load the object into the tool``` is being generated against each Object.

When reviewing the error everything appears to be working correctly.
The validation tool points to the DEV.xml file (http://devwww.openmobilealliance.org/wp/OMNA/LwM2M/DDF.xml) which contains valid DDF and LWM2M element to each of the objects. 

Could you review this issue to see where the disconnect be?

**Comment by SvanteAlnas** [comment]
I receive error 280 only once, in object 11 and there the LWM2M Element is:
"http://www.openmobilealliance.org/wp/OMNA/LwM2M/DDF.xml"

And that is incorrect, the URL should be the URL to the Editor with some parameters to load the editor with the object. The only valid URL to the Editor must start with:
http://devtoolkit.openmobilealliance.org/OEditor/LWMOView?url=......

So this is correct that it is an error that needs to be fixed in the configuration for object ID 11. If you need an example on valid format use another object that is valid, for example the next one:
**http://devtoolkit.openmobilealliance.org/OEditor/LWMOView?url=**http%3A%2F%2Fwww.openmobilealliance.org%2Ftech%2Fprofiles%2FLWM2M_WLAN_connectivity4-v1_0.xml

Please update the wiki/confluence pages with description how to fix this error it so you know it to the next time

**Comment by jpradocueva** [comment]
@seanmcilroy29 the link in the "LWM2M" element for object 11 (in Production) seem incorrect but it is working!!

It should be: 
http://devtoolkit.openmobilealliance.org/OEditor/LWMOView?url=http%3A%2F%2Fwww.openmobilealliance.org%2Ftech%2Fprofiles%2FLWM2M_APN_connection_profile-v1_0.xml

when we did the update last week, it is clear that i forgot to add the first part of the link to open the tool editor.

However, when I look to the production content I see:

```
<LWM2M>http%3A%2F%2Fwww.openmobilealliance.org%2Ftech%2Fprofiles%2FLWM2M_APN_connection_profile-v1_0.xml</LWM2M>
```
And it is working! How is possible?

**Comment by SvanteAlnas** [comment]
That's correct, an parameter in an URL request must be URL encoded so it is possible to parse what is the protocol, domain name, path, query parameters etc.

As described earlier, please don't close this issue until:
**_Please update the wiki/confluence pages with description on how to identify the issue and how to fix this error so you know it to the next time_**
